src/ocrd/cli/log.py

```diff
@@ -18,10 +18,13 @@ def log(self, lvl, *args, **kwargs):
         logger = getLogger(self.name)
         logger.log(getLevelName(lvl), *args, **kwargs)
 
+
 pass_log = click.make_pass_decorator(LogCtx)
 
+
 @click.group("log")
-@click.option('-n', '--name', envvar='OCRD_TOOL_NAME', default='log_cli', metavar='LOGGER_NAME', help='Name of the logger', show_default=True)
+@click.option('-n', '--name', envvar='OCRD_TOOL_NAME', default='log_cli', metavar='LOGGER_NAME',
+              help='Name of the logger', show_default=True)
```

Comment:
```suggestion
@click.option(
'-n', '--name', envvar='OCRD_TOOL_NAME', default='log_cli', metavar='LOGGER_NAME', help='Name of the logger', show_default=True)
```

How about that convention on all `click` options? That should still be acceptable from the POV of the linter. For `click` commands with more parameters, there will be less leading whitespace on each line. Same for all occurrences.

Reply:
Shouldn't the follow-up line still be indented 4 spaces, though?

IMO these 13 characters are not much of a saving to warrant this pattern in general – most of our lines will be much longer, anyway. I would still favour breaking only when needed, and aligning indentation to the opening parentheses.

Reply:
> Shouldn't the follow-up line still be indented 4 spaces, though?

Yes.